hack/for-go-proj.sh

```diff
@@ -18,7 +18,7 @@ set -o errexit
 set -o pipefail
 set -o nounset
 
-CONTRIB_ROOT="$(dirname ${BASH_SOURCE})/.."
+CONTRIB_ROOT="$(dirname "${BASH_SOURCE[0]}")/.."
```

Comment:
I'm not sure that the extra quotation marks here help any, the whole variable is already surrounded by quotes.
Similar issue is visible in kube-env.sh.